docs/superpowers/plans/2026-05-14-party-participants-list.md

```
@@ -0,0 +1,1133 @@
+
+```bash
+git push origin feature/party-participants-list
+```
+
+Expected: 정상 push. PR #149 가 자동으로 새 커밋들 반영.
+
+- [ ] **Step 4: PR 본문 갱신**
+
+PR #149 의 description 에 다음 추가 (또는 새 PR comment):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)